### PR TITLE
dynamic sensor list and sensor dashboard prepared for backend

### PR DESCRIPTION
**Pull request by jluisfgarza**
Hey, the sensor list is currently generated dynamically on the Content.jsx
image

image

Also, the dashboard component is reutilized on each sensor and prepared to fill with new data on Dashboard.jsx
image

image
image

Also I fixes a lot of the issues with the charts components and styled the nav and sidebar.

### CODE REVIEW

**Comment by jarhun88** [comment]
We don't want the axis titles anymore? =(

**Comment by jluisfgarza** [comment]
> We don't want the axis titles anymore? =(

I left that issue open. But I changed the chart.jsx and historychart.jsx so probably you should check the pr and add your changes to it!

File: electron/electron.js [hunk 1]
```diff
@@ -1,135 +1,139 @@
...
+  console.log('[ INFO ] Creating Window')
+  // Create the browser window.
+  windows.mainWindow = new BrowserWindow({
+    width: 900,
+    height: 680,
+    show: false
+  });
+
+  BrowserWindow.addDevToolsExtension(
+    "C:/Users/jluis/AppData/Local/Google/Chrome/User Data/Default/Extensions/fmkadmapgofadopljbjfkapdkoienihi/3.4.2_0"
```
Comment (febg) [inline electron/electron.js]: @jluisfgarza please fix this absolute path to the resource

Comment (jluisfgarza) [reply electron/electron.js]: fixed

File: electron/electron.js [hunk 2]
```diff
@@ -1,135 +1,144 @@
...
+});
+
+/**
+ * window-all-closed: Emitted when all windows have been closed.
+ * This will close the app on all platforms, except Mac,
+ * which will only close the window but will explicitly require
+ * the user to quit the program.
+ * */
+app.on('window-all-closed', () => {
+  if (process.platform !== 'darwin') {
```
Comment (jluisfgarza) [inline electron/electron.js]: Changed if statement. Windows process was not ending correctly.

**Pull request merged**